ansible/roles/blackbox_exporter/templates/blackbox_exporter.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Prometheus Blackbox Exporter
+
+[Service]
+User=nobody
+Group=nogroup
+EnvironmentFile=-/etc/default/blackbox_exporter
+PermissionsStartOnly=true
+ExecStartPre=/sbin/setcap 'cap_net_raw+ep' {{ blackbox_exporter_path }}/blackbox_exporter
```

Comment:
Yeah it's needed for ICMP ping as well as for the DNS based probes. The docs of blackbox_exporter mention these as the [recommended settings](https://github.com/prometheus/blackbox_exporter#permissions).
Maybe @SuperQ knows more.